pages/glossary.md

```diff
@@ -0,0 +1,157 @@
+
+See also:
+- [Relay Kit documentation](./sdk-relay-kit) on docs.safe.global
+
+## Network
+
+A blockchain network is a collection of interconnected computers that utilize a blockchain protocol for communication.
+Decentralized networks allow users to send transactions, that are processed on a distributed ledger with a consensus mechanism ensuring the batching,
+verification, and acceptance of data into blocks. This structure enables the development of applications without the need for a central authority or server.
+
+See also:
+- [Networks](https://ethereum.org/en/developers/docs/networks) on ethereum.org
+
+## Owner
+
+A Safe owner is one of the accounts that control a given Safe, being the only ones that can manage the configuration of a Safe and approve transactions. They can be either [externally-owned accounts](#externally-owned-account) or [smart accounts](#safe-smart-accounts). The [threshold](#threshold) of a Safe defines how many owners need to approve a Safe transaction to make it executable.
+
+See also:
+- [OwnerManager.sol](https://github.com/safe-global/safe-smart-account/blob/main/contracts/base/OwnerManager.sol) on github.com
+
+## Relayer
+
+A relayer is a third-party service acting as an intermediary between users' accounts and [blockchain networks](#network). It executes transactions on behalf of users and covers the associated execution costs, which may or may not be claimed.
+
+See also:
+- [What's Relaying?](https://docs.gelato.network/developer-services/relay/what-is-relaying) on docs.gelato.network
+
+## Safe{DAO}
+
+The Safe{DAO} is the [Decentralized Autonomous Organization](https://ethereum.org/dao) (DAO) that aims to foster a vibrant ecosystem of applications and wallets leveraging Safe accounts. This will be achieved through data-backed discussions, grants, and ecosystem investments, as well as providing developer tools and infrastructure.
+
+See also:
+- [Safe{DAO} Forum](https://forum.safe.global)
+- [Safe{DAO} Governance process](https://forum.safe.global/t/how-to-safedao-governance-process/846) on forum.safe.global
+- [Safe{DAO} Proposals](https://snapshot.org/#/safe.eth) on snapshot.org
+
+## Safe{Wallet}
+
+[Safe{Wallet}](https://app.safe.global) is the official user interface to manage Safe accounts.
+
+See also:
+- [Getting Started with Safe{Wallet}](https://help.safe.global/en/collections/9801-getting-started) on help.safe.global
+
+## Safe Apps
+
+Safe Apps are web applications that run in the Safe Apps marketplace. They support Safe, use the Safe Apps SDK to interact with it, and aren't owned, controlled, maintained, or audited by Safe.
+
+See also:
+- [Safe Apps SDK documentation](./apps-sdk-overview) on docs.safe.global
```

Comment:
```suggestion
- [Safe Apps SDK documentation](./apps-sdk-overview.md) on docs.safe.global
```
Need to add .md here as it will be picked up by linkchecker as a broken link